packages/core/__tests__/cli/check.test.ts

```diff
@@ -176,7 +176,7 @@ describe('CLI: single-pass typechecking', () => {
     `);
   });
 
-  test.skip('reports diagnostics for a companion template type error', async () => {
+  test.only('reports diagnostics for a companion template type error', async () => {
```

Comment:
This test is failing for strange reasons, but might be specific to CLI tests, so not focusing on it for now. Root cause seems to be that the ember component `.ts` file is not ever getting passed to `createVirtualCode` the language plugin.